src/routes/volume/VolumeActions.js

```diff
@@ -5,6 +5,42 @@ import { DropOption } from '../../components'
 const confirm = Modal.confirm
 
 function actions({ selected, engineImages, showAttachHost, detach, showEngineUpgrade, deleteVolume, showBackups, showSalvage, rollback, showUpdateReplicaCount, showExpansionVolumeSizeModal, showCancelExpansionModal, createPVAndPVC, changeVolume, confirmDetachWithWorkload, commandKeyDown }) {
+  const deleteWranElement = (record) => {
+    let workloadStatus = ''
+    let hasPvTootipText = ''
+    let pvStatus = ''
+
+    if (record && record.kubernetesStatus && record.kubernetesStatus.pvStatus) {
+      hasPvTootipText = (<div>{`The longhorn-manager will delete the Kubernetes resources associated with the Volume ${record.name}`}</div>)
+      pvStatus = (<div style={{ marginTop: '10px', color: '#b7b7b7' }}>
+        <div style={{ paddingLeft: '10px' }}>PV Name: {record.kubernetesStatus.pvName}</div>
+        <div style={{ paddingLeft: '10px' }}>PVC Name: {record.kubernetesStatus.pvcName}</div>
+        <div style={{ paddingLeft: '10px' }}>PV Status: {record.kubernetesStatus.pvStatus}</div>
```

Comment:
The `PV Status` shouldn't be needed I think since it's not an actual resource that gets deleted.